docker/genericsql/build.conf

```diff
@@ -1 +1 @@
-version=1.1.0
+version=1.1.0
```

Comment:
What is the change here

Reply:
There was an extra line and I deleted it